src/actions/arcActions.ts

```diff
@@ -1063,10 +1063,15 @@ export function onReputationChangeEvent(avatarAddress: string, address: string)
   }
 }
 
-export function onProposalExecuted(avatarAddress: string, proposalId: string, executionState: ExecutionState, decision: VoteOptions, reputationWhenExecuted: number) {
+export function onProposalExecuted(avatarAddress: string, proposalId: string, decision: VoteOptions, reputationWhenExecuted: number) {
   return async (dispatch: Dispatch<any>, getState: () => IRootState) => {
     const proposal = getState().arc.proposals[proposalId];
 
+    const contributionRewardInstance = await Arc.ContributionRewardFactory.deployed();
+    const votingMachineAddress = (await contributionRewardInstance.getSchemeParameters(avatarAddress)).votingMachineAddress;
+    const votingMachineInstance = await Arc.GenesisProtocolFactory.at(votingMachineAddress);
+    const executionState: ExecutionState = await votingMachineInstance.getProposalExecutionState(proposalId);
```

Comment:
not needed if you are using `GP.ExecutedProposals`

Reply:
https://github.com/daostack/alchemy/pull/122/commits/4744d57fded5c62ebdbe4de258a0eafb39baf33a